tests/integration/control/pod/conftest.py

```diff
@@ -0,0 +1,149 @@
+import pytest
+import random
+import string
+import time
+from pinecone import Pinecone, PodSpec
+from ...helpers import generate_index_name, get_environment_var
+
+@pytest.fixture()
+def client():
+    api_key = get_environment_var('PINECONE_API_KEY')
+    return Pinecone(
+        api_key=api_key,
+        additional_headers={'sdk-test-suite': 'pinecone-python-client'}
+    )
+
+@pytest.fixture()
+def environment():
+    return get_environment_var('PINECONE_ENVIRONMENT')
+
+@pytest.fixture()
+def dimension():
+    return int(get_environment_var('DIMENSION'))
+
+@pytest.fixture()
+def create_index_params(index_name, environment, dimension, metric):
+    spec = { 
```

Comment:
Interesting, I thought you had to use the `PodSpec` class for `spec` rather than passing a plain object.